packages/switchboard/inbound/mailgun.ts

```diff
@@ -1,9 +1,46 @@
 import * as express from "express";
+import { createIncomingEmail, mockCreateIncomingEmail, Email } from "outpostr";
+
+interface InputParams {
+  timestamp: number;
+  token: string;
+  signature: string;
+
+  recipient: string;
+  sender: string;
+
+  "body-mime": string;
+}
+
+function toOutpost(input: InputParams): Email {
+  return {
+    envelope: {
+      from: input.sender,
+      to: [input.recipient],
+    },
+    message: {
+      data: input["body-mime"],
+    },
+  };
+}
 
 export function mailgunHandler() {
-  return function(req: express.Request, res: express.Response) {
-    console.log(req.headers);
-    console.log(req.body);
-    res.send({ok: true});
+  let forwardFunc : any;
+  if (process.env.NODE_ENV == 'test') {
+    forwardFunc = mockCreateIncomingEmail;
```

Comment:
I would be nicer to have API that looks like: `mock.createIncomingEmail`. In other words place all mock functions under mock namespace.